__tests__/app.test.js

```diff
@@ -107,13 +107,49 @@ describe("/api/reviews", () => {
     })
 })
 
-// describe("/api/reviews/:review_id/comments", ()=> {
-//     test ("GET - status 200 - responds with an array of comments for the given review id, each having properties comment_id, votes, created_at, author, body, review_id and sorted by date", ()=>{
-//         return request(app)
-//         .get("/api/reviews/3/comments")
-//         .expect(200)
-//     })
-// })
+describe("/api/reviews/:review_id/comments", ()=> {
+    test ("GET - status 200 - responds with an array of comments for the given review id, each having properties comment_id, votes, created_at, author, body, review_id and sorted by date", ()=>{
+        return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then((res)=>{
+          expect(res.body.comments).toBeSortedBy("created_at", {descending: true})
+          res.body.comments.forEach(comment => {
```

Comment:
Don't forget to assert how many comments we are expecting if we're going to do a forEach.